api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -247,22 +251,40 @@ public void createAllOfUsBillingProject(String projectName, String servicePerime
             .privateIpGoogleAccess(true)
             .servicePerimeter(servicePerimeter);
 
-    BillingApi billingApi = billingApiProvider.get();
-    retryHandler.run(
-        (context) -> {
-          billingApi.createBillingProjectFull(request);
-          return null;
-        });
+    if(isFireCloudBillingV2ApiEnabled()) {
+      BillingV2Api billingV2Api = billingV2ApiProvider.get();
+      retryHandler.run(
+          (context) -> {
+            billingV2Api.createBillingProjectFullV2(request);
+            return null;
+          });
+    } else {
+      BillingApi billingApi = billingApiProvider.get();
+      retryHandler.run(
+          (context) -> {
+            billingApi.createBillingProjectFull(request);
+            return null;
+          });
+    }
   }
 
   @Override
   public void deleteBillingProject(String billingProject) {
-    BillingApi billingApi = billingApiProvider.get();
-    retryHandler.run(
-        (context) -> {
-          billingApi.deleteBillingProject(billingProject);
-          return null;
-        });
+    if(isFireCloudBillingV2ApiEnabled()) {
+      BillingV2Api billingV2Api = billingV2ApiProvider.get();
+      retryHandler.run(
+          (context) -> {
+            billingV2Api.deleteBillingProject(billingProject);
+            return null;
+          });
+    } else {
+      BillingApi billingApi = billingApiProvider.get();
+      retryHandler.run(
+          (context) -> {
+            billingApi.deleteBillingProject(billingProject);
```

Comment:
just verifying, it is OK to use either of these delete calls for any workspace right? It's not the case that we need to call billingV2Api if the workspace was created using billingV2Api and vice versa for billingApiV1.